api/ratelimit/service/ratelimit/v3/rls_conf_ds.proto

```diff
@@ -0,0 +1,25 @@
+syntax = "proto3";
+
+package ratelimit.service.ratelimit.v3;
+
+import "envoy/service/discovery/v3/discovery.proto";
+
+option java_package = "io.envoyproxy.ratelimit.service.config.v3";
+option java_outer_classname = "RlsConfigProto";
+option java_multiple_files = true;
+option go_package = "github.com/envoyproxy/go-control-plane/ratelimit/service/config/v3;configv3";
+option java_generic_services = true;
+
+// [#protodoc-title: Rate Limit Config Discovery Service (RLS Conf DS)]
+
+// Return list of all rate limit configs that rate limit service should be configured with.
+service RateLimitConfigDiscoveryService {
```

Comment:
I don't think we need a new RPC service for this.  I believe we've agreed that as a general strategy for the xDS ecosystem, we're trying to move away from having an individual RPC service for every resource type.  Instead, we should just use ADS for whatever resource types we want.

Reply:
Yeah using ADS sounds like the right path to me, then we don't need much extra work here.